main_test.go

```diff
@@ -0,0 +1,128 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"testing"
+
+	. "github.com/smartystreets/goconvey/convey"
+	"github.com/wtsi-hgi/backup-plans/ibackup"
+	ibackup_test "github.com/wtsi-hgi/backup-plans/internal/ibackup"
+	"github.com/wtsi-hgi/backup-plans/internal/plandb"
+	"github.com/wtsi-hgi/backup-plans/internal/testirods"
+)
+
+const (
+	app = "backup-plans"
+)
+
+func TestMain(t *testing.T) {
+	tmpDir, cleanup := buildSelf()
+	if cleanup != nil {
+		defer cleanup()
+	}
+
+	mysqlConnection := os.Getenv("BACKUP_PLANS_TEST_MYSQL")
```

Comment:
Can use the same env var for tests as previously, and manually check if its a MYSQL one for this purpose